src/main/java/com/minecolonies/coremod/items/ItemAncientTome.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+
+/**
+ * Class handling the AncientTome item.
+ */
+public class ItemAncientTome extends AbstractItemMinecolonies
+{
+    /* default */ private static final int MAX_STACK_SIZE = 64;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)